internal/test/e2e/host.go

```diff
@@ -240,7 +240,7 @@ func NewHost(privKey types.PrivateKey, dir string, network *consensus.Network, g
 		GenesisID:  genesisBlock.ID(),
 		UniqueID:   gateway.GenerateUniqueID(),
 		NetAddress: l.Addr().String(),
-	})
+	}, syncerOpts...)
```

Comment:
Tbh I probably wouldn't have bothered storing the configs on the cluster for reboots and passing in syncer options to `NewHost` and instead just passed hardcoded syncer options here. 
e.g. `}, syncer.WithPeerDiscoveryInterval(testBusCfg().SyncerPeerDiscoveryInterval), syncer.WithSyncInterval(testBusCfg().SyncerSyncInterval))`

I don't think we need to configure tests with specific syncer intervals for hosts in our integration test suite.

Reply:
Hm I considered that too but I figured storing the config also gets rid of a couple of fields. I hardcoded the syncer options in `NewHost` but left the `busCfg` and `apCfg` on the `TestCluster` for now? Don't mind reverting that though.

Reply:
I reverted it anyway, went back and fourth on it while writing it so probably best to specify exactly what is needed from the ap and bus cfg for reboots.